store/migrations/migration0/migrate.go

```diff
@@ -23,288 +15,32 @@ func (m Migration) Timestamp() string {
 }
 
 func (m Migration) Migrate(orm *orm.ORM) error {
-	orm.InitializeModel(&JobSpec{})
-	orm.InitializeModel(&JobRun{})
-	orm.InitializeModel(&Initiator{})
-	orm.InitializeModel(&Tx{})
-	orm.InitializeModel(&TxAttempt{})
-	orm.InitializeModel(&BridgeType{})
-	orm.InitializeModel(&IndexableBlockNumber{})
-	orm.InitializeModel(&User{})
-	orm.InitializeModel(&Session{})
-	orm.InitializeModel(&ServiceAgreement{})
-	return nil
-}
-
-type Time struct {
-	time.Time
-}
-
-// UnmarshalJSON parses the raw time stored in JSON-encoded
-// data and stores it to the Time field.
-func (t *Time) UnmarshalJSON(b []byte) error {
-	var n json.Number
-	if err := json.Unmarshal(b, &n); err != nil {
-		return err
-	}
-	newTime, err := dateparse.ParseAny(n.String())
-	t.Time = newTime.UTC()
+	// orm.DB.LogMode(true) to trace sql commands
+	var err error
+	err = multierr.Append(err, migrationHelper(orm, &models.JobSpec{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.TaskSpec{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.JobRun{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.TaskRun{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.RunResult{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.Initiator{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.Tx{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.TxAttempt{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.BridgeType{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.IndexableBlockNumber{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.User{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.Session{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.Encumbrance{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.ServiceAgreement{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.BulkDeleteRunTask{}))
+	err = multierr.Append(err, migrationHelper(orm, &models.BulkDeleteRunRequest{}))
```

Comment:
I think this can be addressed when we have to write our first migration, but we should at least talk about it sooner: We should probably separate out the expected schema from the actual codebase, otherwise someone who runs the migrations after we've updated a model will have their database in a different state then someone who ran the migrations before we updated the model.

Reply:
Yeah absolutely. I thought we would leave that for another PR. But if you want something else, let me know.